src/data/pages/get-started/index.js

```diff
@@ -931,6 +931,130 @@ const data = {
       title: 'Meet some of our community members',
     },
   },
+  'tw-cn': {
+    hero: {
+      title: '開始了解 eBPF',
+      items: [
+        {
+          image: 'lab',
+          title: '動手實驗',
+          description:
+            '借用 Liz Rice 報告中的 opensnoop 範例，本實驗將教您如何使用 eBPF 工具，觀察其元件載入過程，以及將您自己的追蹤用例添加到 eBPF 原始碼中。',
+          cardColorClassName: 'border-secondary-blue-2',
```

Comment:
I would suggest the following, by analogy with the rest of the language:

```suggestion
          cardColorClassName: 'bg-secondary-blue-1-light border-secondary-blue-1',
          dashedBorderClassName: 'border-secondary-blue-2',
```